src/Microsoft.Framework.Runtime.Interfaces/IRuntimeEnvironment.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Framework.Runtime
+{
+    public interface IRuntimeEnvironment
+    {
+        string OperatingSystem { get; }
+
+        string OperatingSystemVersion { get; }
+
+        string FrameworkFlavor { get; }
+
+        string FrameworkArchitecture { get; }
```

Comment:
I'll go with RuntimeArchitecture because I don't want anyone to confuse it with OS architecture